src/dpapimkv1_fmt_plug.c

```diff
@@ -0,0 +1,436 @@
+/* DPAPI masterkey file version 1 cracker. 
+ * Original version by Dhiru Kholia <dhiru.kholia at gmail.com>.
+ * Reworked and added support of domain credentials by
+ * Fist0urs <jean-christophe.delaunay at synacktiv.com>
+ *
+ * This software is Copyright (c) 2013, Dhiru Kholia <dhiru.kholia at gmail.com>,
+ * Copyright (c) 2017 Fist0urs <jean-christophe.delaunay at synacktiv.com>,
+ * and it is hereby released to the general public under the following terms:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ *
+ * All credits for the algorithm go to "dpapick" project,
+ * https://bitbucket.org/jmichel/dpapick
+ */
+
+#if FMT_EXTERNS_H
+extern struct fmt_main fmt_DPAPImkv1;
+#elif FMT_REGISTERS_H
+john_register_one(&fmt_DPAPImkv1);
+#else
+
+#include <string.h>
+#include <assert.h>
+#include <errno.h>
+#include "arch.h"
+#include "misc.h"
+#include "common.h"
+#include "formats.h"
+#include "params.h"
+#include "memory.h"
+#include "johnswap.h"
+#include "options.h"
+#include "unicode.h"
+#include "sha.h"
+#include "md4.h"
+#include "simd-intrinsics.h"
+#include "hmac_sha.h"
+#define DPAPI_CRAP_LOGIC
+#include "pbkdf2_hmac_sha1.h"
+#include <openssl/des.h>
+#ifdef _OPENMP
+#include <omp.h>
+#ifndef OMP_SCALE
+#define OMP_SCALE               64
+#endif
+#endif
+#include "memdbg.h"
```

Comment:
nitpick: Please include all system headers, then a blank line, the all local ones.